widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidget.kt

```diff
@@ -44,6 +45,6 @@ internal class EntryWidgetImpl(
     }
 
     override fun hide() {
-        TODO("Will be implemented later, in scope of EntryWidget feature")
+        entryWidgetHideController.onHide.onNext(Unit)
```

Comment:
NIT: Could you please move this into function `entryWidgetHideController.hide()`

Reply:
Good idea!

Reply:
Could you please have a look if it's what you meant 3ca36afdfa7ae73da7c6c028729e56123fcc2235